client/src/produkter/variants/VariantsTab.tsx

```diff
@@ -85,18 +79,41 @@ const VariantsTab = ({
 
   const anyExpired = products.some((product) => product.registrationStatus === "INACTIVE");
 
+  const setAsExpired = (product: ProductRegistrationDTO) => {
+    const productRegistrationUpdated: ProductRegistrationDTO = {
+      ...product,
+      registrationStatus: "INACTIVE",
+      expired: todayTimestamp(),
+    };
+
+    updateProductVariant(loggedInUser?.isAdmin || false, productRegistrationUpdated)
+      .then(() => mutateVariants())
+      .catch((error) => {
+        setGlobalError(error);
+      });
+  };
+
+  const setAsActive = (product: ProductRegistrationDTO) => {
+    const productRegistrationUpdated: ProductRegistrationDTO = {
+      ...product,
+      registrationStatus: "ACTIVE",
+      expired: tenYearsFromTodayTimestamp(),
+    };
```

Comment:
Burde jeg lage egne endepunkt til dette sånn som for serie? :)

Reply:
Ja jeg tenker det, fint å være litt konsekvente